dist/button/ds6/button.css

```diff
@@ -204,6 +204,30 @@ a.cta-btn--purple:not([href]) {
   border-color: #c7c7c7;
   color: #f5f5f5;
 }
+button.btn.btn--pill,
+button.expand-btn--pill,
+a.cta-btn--pill,
+a.fake-btn--pill {
+  border-radius: 24px;
+  height: 48px;
+  max-width: 180px;
+  min-width: 80px;
+  padding: 0 24px;
+  vertical-align: top;
+}
+.btn--pill > .btn__cell > span,
+.expand-btn--pill > .expand-btn__cell > span,
+.cta-btn--pill > .cta-btn__cell > span,
+.fake-btn--pill > .fake-btn__cell > span {
+  max-height: 36px;
+  white-space: normal;
+}
+.btn--pill + .btn--pill,
+.expand-btn--pill + .expand-btn--pill,
+.cta-btn--pill + .cta-btn--pill,
+.fake-btn--pill + .fake-btn--pill {
+  margin-left: 8px;
```

Comment:
@yomed @ianmcburnie  What are the chances that different kinds of pills will be used in the same pill set?  This `margin-left` is only applied if the same pill types are adjacent.  If we have a `btn--pill` followed by a `fake-btn--pill` then the margin isn't applied.

What about using:
```
.btn + .btn--pill
.btn + .expand-btn--pill
etc.
```

Reply:
![screen shot 2018-09-19 at 5 28 08 pm](https://user-images.githubusercontent.com/1562843/45788781-6b096f80-bc31-11e8-90ff-0d7a50feb29c.png)

Reply:
What about:

```css
[class*="btn--pill"] + [class*="btn--pill"] {
  margin-left: 8px;
}
```

Would that capture all cases? Please bear in mind I've been away 6 weeks ;-)

Reply:
Sure, looks fine